src/components/ThoughtList/Thought.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import style from './ThoughtList.css'
+
+export default function Thought({ thought, onChange, onDelete }) {
+  const [beingEdited, setBeingEdited] = useState(false);
+
+  let thoughtContent
+
+  if(beingEdited) {
+
+    thoughtContent = (
+      <div>
+        <input 
+        type="text" 
+        className={style.EditInput}
+        placeholder={thought.text}
+        onChange={(e) => {
+          onChange({
```

Comment:
it would be better to have a text piece of state that the onChange updates instead of updating state with every letter the user types.